### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by samuelito22**
Here are my beautiful commits made with `git-commit-plugin` that you can find in the github extension market. Below, there is a higher-level description of the extension. Look at the commits I made, learn from them. If you notice something is wrong let me know, as well as for any queries you may have. This branch will not be deleted even after the branch has been merged following this pull request, this is because I need to now create the authenticaiton page for the client folder.

Name: git-commit-plugin
Id: redjue.git-commit-plugin
Description: Automatically generate git commit
Version: 1.4.0
Publisher: redjue
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=redjue.git-commit-plugin

### CODE REVIEW

File: server/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import morgan from 'morgan'
+import { config } from './shared/config'
+import UserRoutesV1 from './modules/user/v1/routes'
+
+const app = express()
+
+// Middleware
+app.use((req, res, next) => {
```
Comment (MichaelK999) [inline server/src/app.ts:10]: Stripe middleware

File: server/src/modules/user/v1/models/Feedback.model.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  PrimaryKey,
+  Table,
+} from 'sequelize-typescript'
+import { User } from './User.model'
+import { Optional } from 'sequelize'
+import { IFeedback } from '../types'
+
+type FeedbackCreationAttributes = Optional<IFeedback, 'createdAt' | 'id'>
+
+@Table({
```
Comment (MichaelK999) [inline server/src/modules/user/v1/models/Feedback.model.ts:17]: All this stuff I am new to, is this the ORM & should I do a tutorial on this?

File: server/src/modules/user/v1/models/UserProfile.model.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import {
```
Comment (MichaelK999) [inline server/src/modules/user/v1/models/UserProfile.model.ts:1]: Lots of ORM stuff I am unsure of

File: server/src/modules/user/v1/types/models.type.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
+export interface IUser {
```
Comment (MichaelK999) [inline server/src/modules/user/v1/types/models.type.ts:1]: What is this? User, IUserProfile etc

File: server/src/modules/user/v1/validators/user.validator.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from 'express'
+import Joi from 'joi'
+
+const userProfileSchema = Joi.object({
```
Comment (MichaelK999) [inline server/src/modules/user/v1/validators/user.validator.ts:4]: What is Joi, and I thought ( look at other comments ) that we already created the schema

**Review by MichaelK999 — APPROVED** [review]
Nice job! Some stuff I didn't fully understand so have asked questions about.

File: server/src/modules/user/v1/models/Feedback.model.ts (hunk 2 above)
Comment (samuelito22) [reply server/src/modules/user/v1/models/Feedback.model.ts:17]: Yes, you sequelize-typescript for ease.

File: server/src/modules/user/v1/validators/user.validator.ts (hunk 5 above)
Comment (samuelito22) [reply server/src/modules/user/v1/validators/user.validator.ts:4]: Joi is for validation. Is the same as typeof name === "string" and so on

**Pull request merged**